### PR TITLE
LPS-144712

### PR DESCRIPTION
**Pull request by rodrigopaulino**
Related Issue: https://issues.liferay.com/browse/LPS-144712

### CODE REVIEW

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/portlet/DDMFormAdminPortlet.java [hunk 1]
```diff
@@ -122,6 +128,13 @@ public void render(
 		super.render(renderRequest, renderResponse);
 	}
 
+	@Activate
+	protected void activate(Map<String, Object> properties) {
+		_ffDateTimeDDMFormFieldTypeConfiguration =
+			ConfigurableUtil.createConfigurable(
+				FFDateTimeDDMFormFieldTypeConfiguration.class, properties);
+	}
+
```
Comment (carolmariaabb) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/portlet/DDMFormAdminPortlet.java:137]: @rodrigopaulino why didn't you create a `FFDateTimeDDMFormFieldTypeConfigurationActivator`? Can we reuse this logic when creating a new feature flag in another story?

**Comment by rodrigopaulino** [comment]
ci:stop

**Comment by rodrigopaulino** [comment]
Forgot to add test mocks.

**Comment by liferay-forms** [comment]
Hey @rodrigopaulino, can you please check if these failures are related to your changes? Thanks

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/portlet/DDMFormAdminPortlet.java (hunk 1 above)
Comment (rodrigopaulino) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/portlet/DDMFormAdminPortlet.java:137]: Sure we can. The "Activator" component is only made necessary when there's already an `activate` method in the class and you can't get the properties from it. See [this](https://github.com/liferay-forms/liferay-portal/blob/15ee083a549625186a875cd53bef8731c6a44974/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/java/com/liferay/dynamic/data/mapping/form/renderer/internal/DDMFormTemplateContextFactoryImpl.java#L156) example. The only way to inject a configuration in this class is by adding a `@Reference` to another component that loads the configuration.
Does it make sense?

**Comment by rodrigopaulino** [comment]
ci:test:sf

**Review by carolmariaabb — APPROVED** [review]

**Comment by rodrigopaulino** [comment]
ci:forward

**Pull request closed without merging**